src/administration/integrations/gluescript.md

```diff
@@ -0,0 +1,249 @@
+# GlueScript
+
+GlueScript scripts are powered by a lightweight Javascript engine that allows you to run a script on any activity within Platform.sh. This opens a whole world of new automation use-cases including, but not limited to, calling external web services.
+
+This is not a full-blown Javascript runtime, and you can't use NPM or import libraries. Keep it short and sweet.
+
+This is very much an **Alpha release**, and while the Javascript API itself will probably be stable enough, the script management will undoubtedly change. For the moment you would simply post a `.js` file to our REST API to create the integration. 
+
+> **CAVEAT**: Currently errors are silent so this is not for the faint of heart.
+
+While you don't get to have NPM GlueScript comes with a bunch of helpers that make it very useful. You can currently access the following APIs: [Project](#project-api), [Activity](#activity-api), [Process](#process-api), [Storage](#storage-api), [Fetch](#crypto-api) and [Crypto](#crypto-api)
+
+# Setting up a GlueScript integration
+
+## 1. Write the script
+
+Create a new `scripts` folder within the `.platform/` folder.
+
+```bash
+mkdir .platform/scripts
+```
+
+Create a new Javascript file inside the `scripts` folder with the following content:
+
+```javascript
+{
+  "type": "script",
+  "script": "<CODE>"
+}
+```
+
+where:
+* `CODE` is the Javascript code of the integration
+
+#### Custom Slack integration
+
+For example, the GlueScript can perform a custom Slack integration as follows:
+
+```javascript
+var message = activity.text;
+var color;
+
+if (activity.state == 'complete') {
+  if (activity.result == 'success') {
+    color = '#66c000';
+  } else {
+    color = '#ff0000';
+  }
+}
+
+var resp = fetch(
+  '<SLACK_PAYLOAD_URL>',
+  {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json',
+    },
+    body: JSON.stringify({
+      'attachments': [{
+        'fallback': message,
+        'color': color,
+        'fields': [{
+          'value': message,
+        }],
+      }],
+    }),
+  }
+);
+
+if (!resp.ok) { throw 'Failed'; }
+```
+
+where:
+* `SLACK_PAYLOAD_URL` is the URL provided by Slack for your custom webhook
+
+### 2. Commit the script
+
+Commit the script to Git:
+
+```bash
+git add .platform/scripts/SCRIPT.js
+git commit -m "Committing the GlueScript file."
+git push platform master
+```
+
+where:
+* `SCRIPT` is the name of your Javascript file
+
+### 3. Enable the integration
+
+Enable the GlueScript integration as follows (you need to have the Platform.sh CLI installed):
+
+```bash
+platform integration:add --type script --file .platform/scripts/SCRIPT.js
```

Comment:
when using the CLI the file needs to just be JavaScript (not JSON wrapping JavaScript)

Reply:
test with https://github.com/platformsh/platformsh-cli/pull/823#issuecomment-510856921